### ISSUE TITLE
Lab 4

### ISSUE THREAD

**Issue by StephanieDeus** [Watts-College/paf-515-spr-2024#10]
Hi Professor Stowers,

I am encountering an error when I need to filter our data set to exclude tracts without SVI flag data for 2010 and 2020 . I'm not sure why R is not finding the object "pre10_nmtc_project_cnt" when it loaded successfully.

Can you please help me?
<img width="905" alt="Screenshot 2024-04-14 at 3 41 03 AM" src="https://github.com/Watts-College/paf-515-spr-2024/assets/159660259/34e0acca-3b7e-4b02-a499-f2ca382cc125">
<img width="905" alt="Screenshot 2024-04-14 at 3 42 03 AM" src="https://github.com/Watts-College/paf-515-spr-2024/assets/159660259/0f8d3726-a156-45c9-8033-7e0402fae8a7">

**Comment by castower** [comment]
Hello @StephanieDeus,

You will want to make sure you've completed the step to join to the pre2010 data before the post2010 data. 

This error indicates the join didn't save properly.

You can search the code-through for the following line to find the step:

`Find count of pre2010 NMTC projects`

**Comment by StephanieDeus** [comment]
How do I save it properly? I ran the code and it still gives the error and I can't move forward.

**Comment by castower** [comment]
Do you have the code chunk that appears below that header in your .rmd file? @StephanieDeus

**Comment by StephanieDeus** [comment]
Yes

**Comment by StephanieDeus** [comment]
<img width="1431" alt="Screenshot 2024-04-14 at 10 00 54 AM" src="https://github.com/Watts-College/paf-515-spr-2024/assets/159660259/5770fee5-688e-486f-92ea-2207f20fb850">

**Comment by castower** [comment]
@StephanieDeus There should be the step to join the pre2010 data to your svi_divisional_nmtc_eligible data frame before joining the post2010 data.

**Comment by StephanieDeus** [comment]
Yes! Thank you, I looked over the code-through so many times and did not realize that I missed that step. I should put on my glasses. Thank you so much!

**Comment by castower** [comment]
@StephanieDeus 

No worries, it happens all the time!! 

Glad that resolved your issue :)
